relay/blockchain/delegate.py

```diff
@@ -64,7 +65,9 @@ def calc_next_nonce(self, identity_address: str):
         except UnexpectedIdentityContractException:
             raise InvalidIdentityContractException
 
-    def calculate_fees_for_meta_transaction(self, meta_transaction: MetaTransaction):
+    def calculate_fees_for_meta_transaction(
+        self, meta_transaction: MetaTransaction
+    ) -> DelegationFees:
```

Comment:
Something seems wrong here with the type hint. 
I wonder why this is not caught by mypy.

Reply:
If I run `mypy --ignore-missing-imports relay tests` it does not raise any error. But my IDE complains on the line `return self.delegation_fees` by saying "Expected type 'DelegationFees', got 'List[DelegationFees]' instead "

Reply:
It also does not complain when I run `mypy --ignore-missing-imports relay/blockchain/delegate.py `

Reply:
mmh, sad, we could maybe see if we can somehow upgrade or so, to see if that fixes it. 
I'll open an issue.